### ISSUE TITLE
Deploy Local Network

### ISSUE THREAD

**Issue by narayanprusty** [spacemeshos/go-spacecraft#1]
The idea is to deprecate local-testnet tool and deploy containers on minikube using go-spacecraft.

**Comment by avive** [comment]
Q: will the dev experience be on par to what we have achieved with localnet? I think we got to the optimal number of steps to setup a network - 1. install npm package. 2. create & run a network... I think it is important to make sure we can provide good dev experience with it - on same or close level to previous implementation...

**Comment by narayanprusty** [comment]
Yes the idea is the achieve the same user experience as local testnet
